generate.json

```diff
@@ -6,5 +6,9 @@
     {
         "taskName": "S3Download",
         "taskClient": "S3"
+    },
+    {
```

Comment:
This file will be more maintainable if we keep the entries with the same taskClient grouped together.
It will be _even more_ maintainable if we order by taskClient then by taskName, but I'll leave that up to you.

Reply:
I disagree, I think it should just be ordered by TaskName like tasks are

Reply:
I disagree, I think it should just be ordered by TaskName like tasks are, reordered it to be that way